src/addons/Textarea/Textarea.js

```diff
@@ -1,40 +1,19 @@
 import React, {Component, PropTypes} from 'react';
-import Field from 'src/collections/Form/Field';
+import classNames from 'classnames';
 
-class Textarea extends Component {
+export default class Textarea extends Component {
   static propTypes = {
-    dataContent: PropTypes.string,
+    className: PropTypes.string,
     label: PropTypes.string,
-    name: PropTypes.string,
-    onChange: PropTypes.func,
-    ref: PropTypes.string,
-    rows: PropTypes.oneOfType([
-      PropTypes.string,
-      PropTypes.number
-    ]),
-    value: PropTypes.string,
-  };
-
-  static defaultProps = {
```

Comment:
Yep, none of these are required now since we inherit props and classes.  We don't need to define every prop and it's type, since any prop and type you add will be passed down.  We just need to define the ones we need to use.  Like `className` and `label`.

Same for `defaultProps`, this library will be generic and assume no defaults.  If you'd like 3 rows, simply add `rows={3}` to the component. :+1: